GooglePayment/src/test/java/com/braintreepayments/api/GooglePaymentUnitTest.java

```diff
@@ -94,7 +94,7 @@ public void setup() {
         when(GoogleApiAvailability.getInstance()).thenReturn(mockGoogleApiAvailability);
 
         ActivityInfo mockActivityInfo = mock(ActivityInfo.class);
-        when(mockActivityInfo.getThemeResource()).thenReturn(2132083045);
+        when(mockActivityInfo.getThemeResource()).thenReturn(R.style.bt_transparent_activity);
```

Comment:
did this test start failing with the android version bump?

Reply:
It might have I'm honestly not sure. There was a hardcoded integer here though representing one of our style constants. I don't think the Android build system guarantees those to stay the same over time.